### DISCUSSION TITLE
Trumbowyg editor height

### DISCUSSION THREAD

**Discussion by kevinsmgov** [OrchardCMS/OrchardCore#11980] [Q&A]
When I change my HtmlBody editor from wysiwyg to trumbowyg, the default editor height (I believe it's 300px) and scrollbar goes away and the editor expands vertically to include all content (i.e. you can't get to the editor buttons without scrolling the page up). Is this configurable or is it something in the Orchard Stack that needs to be corrected?

**Comment by hishamco** [comment]
Please share a GIF screenshot showing the issue

**Reply by kevinsmgov** [reply]
![Screenshot 2022-07-09 080133](https://user-images.githubusercontent.com/5798763/178111249-a69dd9bc-531b-4e61-83f9-dcf099aee95c.gif)
